Wikipedia/Code/WMFAuthAccountCreationInfoFetcher.swift

```diff
@@ -22,6 +22,17 @@ public class WMFAuthAccountCreationInfo: NSObject {
         self.canCreateAccounts = canCreateAccounts
         self.captchaID = captchaID
         self.captchaURLFragment = captchaURLFragment
+    }    
+    var captchaImageURL: URL? {
+        guard let appLang = MWKLanguageLinkController.sharedInstance().appLanguage else {
+            return nil
+        }
+        let siteURL = appLang.siteURL() as NSURL
+        guard let domain = siteURL.wmf_domain else {
```

Comment:
Hey @joewalsh, 
Will we need to convert NSURL category methods from `#import "NSURL+WMFLinkParsing.h"` to Swift URL extensions before we can use them with Swift URLs?

I added `#import "NSURL+WMFLinkParsing.h"` to `Wikipedia-Bridging-Header.h` but wasn't able to call `wmf_domain` without casting to NSURL...